src/utils/helpers.js

```diff
@@ -46,29 +46,15 @@ export function isAllDayEvent(event, { startAccessor, endAccessor, allDayAccesso
 }
 
 /**
- * Returns a filter function that returns true if an event should be rendered
+ * Returns a filter function that returns true if an event or availability should be rendered
  * in the specified date's day column.
  */
-export function makeEventFilter(date, { startAccessor, endAccessor }) {
-  return function filterDayEvents(event) {
-    const start = get(event, startAccessor);
+export function makeEventOrAvailabilityFilter(date) {
+  return function filterDayEventsOrAvailabilities(eventOrAvailability, startAccessor, endAccessor) {
```

Comment:
Why were `startAccessor` and `endAccessor` moved into the nested function? That makes the nested function a lot clunkier to use `.filter` with (you can't just do `.filter(filterFn)`, you have to do `.filter((item) => filterFn(item, startAccessor, endAccessor))`)

Reply:
@sidneytang-elation  When `startAccessor` and `endAccessor` were left in the parent function, they returned `undefined` on the second call of the parent. `makeEventOrAvailabilityFilter` would work as expected on the first call for events, but `startAccessor` and `endAccessor` returned `undefined` on the second call for availabilities.

Reply:
Can you give an example of the code you're talking about? It's not clear to me why having them in the parent function wouldn't work. Were you only instantiating one filter function and calling it for both events and availabilities or something?

Reply:
OMG... just figure it out typing my response... update incoming